### ISSUE TITLE
Icons not showing up

### ISSUE THREAD

**Issue by Usama129** [sivaprakashDesingu/react-voice-recorder#8]
Hi, everything works fine - except that the Mic icon and the Close icon show their alt text... am I missing a library/CSS?

**Comment by sivaprakashDesingu** [comment]
@Usama129  Could you please share your screen. Seems all icons rendering perfect in my local.

**Comment by nsr-py** [comment]
Facing same issue on chrome as well as firefox

![Chrome](https://user-images.githubusercontent.com/57226514/101812565-85490400-3b41-11eb-87d4-88876e82449d.png)
![Firefox](https://user-images.githubusercontent.com/57226514/101812658-a873b380-3b41-11eb-9452-b2f2ef7ed81d.png)

**Comment by nsr-py** [comment]
You could directly use material-ui icons instead, they look somewhat better and will be available on all browsers (*I guess*), if you want I can build a new prototype with this approach?

**Comment by sivaprakashDesingu** [comment]
Seems there is issue on config file. Now resolved the same issue with **react-voice-recorder@2.0.7**
Please let me know if you have the same issue.
